pkg/security/scc/byrestrictions.go

```diff
@@ -75,10 +78,40 @@ func volumePointValue(scc *kapi.SecurityContextConstraints) int {
 	}
 
 	if hasHostVolume {
-		return 10
+		return 10000
 	}
 	if hasNonTrivialVolume {
-		return 5
+		return 5000
 	}
 	return 0
 }
+
+// hasCap checks for needle in haystack.
+func hasCap(needle kapi.Capability, haystack []kapi.Capability) bool {
+	for _, c := range haystack {
+		if needle == c {
+			return true
+		}
+	}
+	return false
+}
+
+// capabilitiesPointValue returns a score based on the capabilities allowed,
```

Comment:
Could you add more comments about why we need this and what was the logic behind the statements?